docs/tutorial-forms.rst

```diff
@@ -111,6 +111,23 @@ Adding Additional Attributes
 ----------------------------
 
 
+Add Red Required Asterisk
+-------------------------
+This can be accomplished by using native Django and CSS.  To do this, add this attribute to 
+the *Django Form*, which is also shown here in the `Django Docs <https://docs.djangoproject.com/en/1.8/ref/forms/api/#django.forms.Form.required_css_class>`_:
```

Comment:
Be careful, there is a double space between "CSS." and "To do". Moreover, this solution works great from django 0.8. Also, there is a slight difference between 1.8 and previous versions but I'm not sure its worth mentionning it.